README.md

```diff
@@ -46,6 +46,38 @@ fact, without the
 ```
 that's the version that will be used by default.
 
+# Quickstart - Maven Example
+### Step 1: Add the required maven dependencies.
+```xml
+<dependency>
+    <groupId>org.owasp.esapi</groupId>
+    <artifactId>esapi</artifactId>
+    <version>2.5.4.0</version>
+    <classifier>jakarta</classifier>
+</dependency>
+<dependency>
```

Comment:
I'm not sure I would show this dependency. For the older Java EE Servlet API, we have that as:
```xml
        <dependency>
            <groupId>javax.servlet</groupId>
            <artifactId>javax.servlet-api</artifactId>
            <version>3.1.0</version>
            <scope>provided</scope>
        </dependency>
```

Usually,, if you are using Jakarta EE, the Jakarta Servlet API is either pulled in by the JDK itself (I think for versions later than Java 17 that's the case, but I may be wrong about the version number) or your pulling it in through another dependency like Spring Boot 3.x, Spring Framework 6.x, or Tomcat 10.x or later.

The problem with including this dependency is that someone would then expect us to keep this updated to the latest version and since we just convert the byte code for the Jakarta version of ESAPI, we really don't explicitly check compatibility with any particular version of the Jakarta EE Servlet Spec.

Reply:
I'll have to think about this. I don't want or team to have the burden to have to keep dependency versions in the **README.md** in sync with that is reflected in our **pom.xml.**. This may be content I move to our a local GH wiki page. I'd been enough of a PITA just for me to remember to update _ESAPI_ versions in the README. And I can't recall anyone who is using ESAPI with only Java SE. Probably 80% of the ESAPI components require the Servlet API from JavaEE (or JarkataEE, depending on your perspective). And those who start out using ESAPI from the start is likely a minority. Based on both secure code reviews that I've done and feedback from developers from private emails and SO posts, most of them only start using ESAPI when some SAST tool identifies vulnerabilities and suggests ESAPI as a potential solution, and usually in the source-to-sink taint path the follow, the 'source' is most often something like an HTTP parameter associated with the Servlet API.